lua/lazyvim/plugins/extras/lang/elixir.lua

```diff
@@ -6,37 +6,8 @@ return {
     })
   end,
   {
-    "neovim/nvim-lspconfig",
-    opts = {
-      servers = {
-        elixirls = {
-          keys = {
-            {
-              "<leader>cp",
-              function()
-                local params = vim.lsp.util.make_position_params()
-                LazyVim.lsp.execute({
-                  command = "manipulatePipes:serverid",
-                  arguments = { "toPipe", params.textDocument.uri, params.position.line, params.position.character },
-                })
-              end,
-              desc = "To Pipe",
-            },
-            {
-              "<leader>cP",
-              function()
-                local params = vim.lsp.util.make_position_params()
-                LazyVim.lsp.execute({
-                  command = "manipulatePipes:serverid",
-                  arguments = { "fromPipe", params.textDocument.uri, params.position.line, params.position.character },
-                })
-              end,
-              desc = "From Pipe",
-            },
-          },
-        },
-      },
-    },
+    "mason-org/mason.nvim",
+    opts = { ensure_installed = { "expert" } },
```

Comment:
```suggestion
    "neovim/nvim-lspconfig",
    opts = { 
      servers = { 
        expert = {}, 
      },
    },
```
This is more close to the way LazyVim structures the LSP configuration.